src/Query.ts

```diff
@@ -93,6 +93,12 @@ export class Query<T = unknown> implements PromiseLike<T> {
     }
   }
 
+  clear = (): void => {
```

Comment:
my only potential suggestion here is using `clearData` vs. clear just to make it more clear what you are clearing. clear?

Reply:
Ok I thought that was clear but we can rename it.